libiop/bcs/merkle_tree.hpp

```diff
@@ -103,6 +126,8 @@ class merkle_tree {
 
     hash_digest_type get_root() const;
 
+    /* These two functions do not currently work if the given positions aren't sorted or
+       have duplicates, AND the tree is set to be zero knowledge. */
```

Comment:
Is this one a new limitation that is caused by the present PR?

Reply:
I really don't remember. It might be new due to the cap hashes. If so, then the issue will not be present if you don't use the new cap hash optimization.

Reply:
I highly recommend explicitly sorting the positions within this function and lift such limitation, as it is unnatural.